masterlist.yaml

```diff
@@ -3746,6 +3746,15 @@ plugins:
       - crc: 0x38DADE03
         util: 'SSEEdit v4.0.4b'
 
+  - name: 'HagravenSkinShaders.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/16310/'
+        name: 'Realistic Skin Shaders - Falmer and Hagravens'
+    tag: [ Graphics ]
```

Comment:
The mod just edits the alternate textures which aren't currently imported by the graphics patcher.

```suggestion
```